### PR TITLE
(maint) fixup fpm bugs

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb [hunk 1]
```diff
@@ -3,60 +3,61 @@
...
 # and setup needed to build packages for <os>
 # The os string should be one of el, sles, debian, ubuntu,
 # or fedora
 # The version should be the numerical OS version for el,
 # sles, and fedora and the codename for debian/ubuntu.
+# The resulting artifacts will be stored in <basepath>
 ################
 build_os=$1
 build_ver=$2
+basepath=$3
```
Comment (melissa) [inline resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb]: Should we have a default so that folks aren't required to set basepath?

Comment (demophoon) [reply resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb]: oops, i merged already :X

Comment (underscorgan) [reply resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb]: :woman_shrugging: I don't know that this script is ever going to be used manually, it just gets executed from rake tasks

Comment (demophoon) [reply resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb]: That was my understanding of this script

Comment (underscorgan) [reply resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb]: can always fix it later if that usecase ends up happening

**Pull request merged**